src/slices/releaseSlice.test.js

```diff
@@ -0,0 +1,202 @@
+import releases, {
+  getPending,
+  getSuccess,
+  getError,
+  putIsPublicPending,
+  putIsPublicSuccess,
+  putIsPublicError
+} from "./releaseSlice";
+
+console.log(releases);
+
+const initialState = {
+  pending: false,
+  error: "",
+  items: [
+    {
+      "0": {
+        id: 100,
+        productVersion: {
+          id: 100,
+          productId: 100,
+          product: null,
+          version: "2.5.4",
+          isPublic: false
+        },
+        title: "Finally Rich",
+        isPublic: false,
+        releaseNotes: []
+      }
+    }
+  ]
+};
+
+const initialStateWithoutItems = {
+  pending: false,
+  error: "",
+  items: []
+};
+
+describe("releases reducer", () => {
+  it("should handle initial state", () => {
+    expect(releases(undefined, {})).toEqual(initialStateWithoutItems);
+  });
+
+  it("should handle putIsPublicSuccess", () => {
+    expect(
+      releases(initialState, {
+        type: getSuccess.type,
+        payload: {
+          id: 100,
+          productVersion: {
+            id: 100,
+            productId: 100,
+            product: null,
+            version: "2.5.4",
+            isPublic: false
+          },
+          title: "Finally Rich",
+          isPublic: true,
+          releaseNotes: []
+        }
+      })
+    ).toEqual({
+      pending: false,
+      error: "",
+      items: {
+        id: 100,
+        productVersion: {
+          id: 100,
+          productId: 100,
+          product: null,
+          version: "2.5.4",
+          isPublic: false
+        },
+        title: "Finally Rich",
+        isPublic: true,
+        releaseNotes: []
+      }
```

Comment:
Item bør være en array?

Reply:
Eg mein å husk at gjor det på den fordi *items* ikke bi en array, når du kun legg inn et element.

Reply:
Kan lag en tell test som test når du legg inn fleire element.

Reply:
Føle ditta tilfellet blir litt dårlig kode, når du ikkje kan vite om du skal forvente en array eller et objekt. Lett å snuble over errors når du skal ha tak i dataen.

Reply:
Eg skal lag en ny commit

Reply:
Resolved in branch  *feature_get_single_release*